go.mod

```diff
@@ -1,15 +1,13 @@
 module github.com/keep-network/go-libp2p-bootstrap
 
-go 1.12
+go 1.16
 
 require (
-	github.com/ipfs/go-ipfs-config v0.0.4
-	github.com/ipfs/go-log v0.0.1
-	github.com/jbenet/goprocess v0.1.3
-	github.com/libp2p/go-libp2p-host v0.1.0
-	github.com/libp2p/go-libp2p-net v0.1.0
-	github.com/libp2p/go-libp2p-peer v0.2.0
-	github.com/libp2p/go-libp2p-peerstore v0.1.0
-	github.com/libp2p/go-libp2p-routing v0.0.1
-	github.com/libp2p/go-testutil v0.1.0
+	github.com/ipfs/go-ipfs-config v0.16.0
+	github.com/ipfs/go-log v1.0.5
+	github.com/jbenet/goprocess v0.1.4
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/libp2p/go-libp2p-core v0.9.0
```

Comment:
Can we land on https://github.com/libp2p/go-libp2p-core/releases/tag/v0.10.0 ?

Reply:
Nope. The `keep-core` repo uses `go-libp2p v.0.15.1` wchich depends on `v0.9.0` (see: https://github.com/libp2p/go-libp2p/blob/4bc78f1d3a4f2ad30d0e6974d27e5ff8dbf61ef4/go.mod#L26). It doesn't compile with `v0.10.0`. Already tried it.